src/vs/vscode.d.ts

```diff
@@ -5400,6 +5400,26 @@ declare module 'vscode' {
 	}
 
 	/**
+	 * An event describing the change in Configuration
+	 */
+	export interface ConfigurationChangeEvent {
+
+		/**
+		 * Affected configuration keys
+		 */
+		readonly affectedKeys: string[];
+
+		/**
+		 * Returns `true` if the given section for the given resource (if provided) has affected.
+		 *
+		 * @param section Configuration name, supports _dotted_ names.
+		 * @param resource A resource Uri.
+		 * @return `true` if the given section for the given resource (if provided) has affected.
+		 */
+		affectsConfiguration(section: string, resource?: Uri): boolean;
```

Comment:
I find this method name a bit strange -- maybe something like `hasChanged` or `wasChanged` would be a bit clearer? Although having something like:

```ts
getChange(section: string, resource?: Uri) : ({ previous: any, current: any }) | undefined
```
Something like that would be nicer IMO, since often it is important to know the previous value, as well has not having to call a `wasChanged` type of method first, to then go an call into the configuration to get the new value.

Although this method isn't really needed if the above comment is used.